remote-messaging/remote-messaging-store/src/main/java/com/duckduckgo/remote/messaging/store/RemoteMessagingConfigDao.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.remote.messaging.store
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+abstract class RemoteMessagingConfigDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
I am probably confused but here it goes anyway. The `RemoteMessagingConfig` has a primary key that defaults always to `1` and it is not `auto-increment`.
We don't seem to pass the `id` ever when creating `RemoteMessagingConfig`, which means we're always replacing the models.

If the above is true, is that the intention?